doc/release/README.chplenv.rst

```diff
@@ -0,0 +1,555 @@
+
+        export CHPL_HOME=~/chapel-1.12.0
+
+   .. note::
+     This, and all other examples in Chapel READMEs, assumes you're using the
+     'bash' shell.  If using 'csh' or 'tcsh', mentally change 'export FOO=BAR'
+     to 'setenv FOO BAR'.  If using some other shell, make the appropriate
+     adjustment.
+
+
+#. Set the CHPL_HOST_PLATFORM environment variable to represent the platform on
+   which you're working.  For standard UNIX workstations, this can be done by
+   running the $CHPL_HOME/util/chplenv/chpl_platform.py script.  For example::
+
+        export CHPL_HOST_PLATFORM=`$CHPL_HOME/util/chplenv/chpl_platform.py`
+
+   For other platforms that appear very similar to a UNIX workstation from the
+   shell prompt (e.g., a Cray XK (TM)), the value may need to be set
+   explicitly.  The strings for our currently-supported host platforms are as
+   follows:
+   
+        ===========  ==================================
+        Value        Description
+        ===========  ==================================
+        cygwin32     x86 Cygwin (Windows) platforms
+        cygwin64     x86_64 Cygwin (Windows) platforms
+        darwin       Macintosh OS X platforms
+        linux32      32-bit Linux platforms
+        linux64      64-bit Linux platforms
+        marenostrum  BSC's MareNostrum platform
+        netbsd32     32-bit NetBSD platforms
+        netbsd64     64-bit NetBSD platforms
+        pwr5         IBM Power5 SMP cluster
+        pwr6         IBM Power6 SMP cluster
+        sunos        SunOS platforms
+        cray-cs      Cray CS (TM)
+        cray-xc      Cray XC (TM)
+        cray-xe      Cray XE (TM)
+        cray-xk      Cray XK (TM)
+        ===========  ==================================
+
+   Platform-specific READMEs are available for most of these platforms in
+   $CHPL_HOME/doc/platforms.  See $CHPL_HOME/doc/README for a directory of
+   these files.
+
+   The Chapel Makefiles and sources are designed to work for any
+   UNIX-compatible environment that supports a GNU-compatible make utility.
+   The list above represents the set of platforms that we have access to and
+   can test easily.  We are interested in making our code framework portable to
+   other platforms -- if you are using Chapel on a platform other than the ones
+   listed above, please refer to the README.building file and/or contact us at:
+   chapel_info@cray.com
+
+#. Set your PATH to include the directory $CHPL_HOME/bin/$CHPL_HOST_PLATFORM
+   which is created when you build the compiler.  For example::
+
+        export PATH="$PATH":"$CHPL_HOME/bin/$CHPL_HOST_PLATFORM"
+
+
+#. Set your man path to include the directory $CHPL_HOME/man.
+   For example::
+
+        export MANPATH="$MANPATH":"$CHPL_HOME"/man
+
+Optional Settings
+-----------------
+
+CHPL_HOST_PLATFORM
+~~~~~~~~~~~~~~~~~~
+   If you are cross-compiling for a platform other than your
+   $CHPL_HOST_PLATFORM, set the CHPL_TARGET_PLATFORM environment variable to
```

Comment:
Also, since the steps above specifically mention CHPL_HOST_PLATFORM, it's not really "optional", right?  Should the title of this section be CHPL_TARGET_PLATFORM instead?